bin/main.ml

```diff
@@ -144,6 +144,7 @@ let all =
   ; Fmt_cmd.command
   ; Compute.command
   ; Upgrade.command
+  ; Print_targets.command
```

Comment:
Can you add a blackbox test so that we can see how verbose the output it? to do that, you can create a directory with a `run.t` file under `tests/blackbox-tests/test-cases`, run the tests, promote the diff to update dune files, and then the test should run.

Reply:
Done (sorry, completely forgot about this PR)